app/page/index.html

```diff
@@ -1,5 +1,5 @@
 <!DOCTYPE html>
-<!-- Wire, Copyright (C) Wire Swiss GmbH -->
+<!-- Wire, Copyright (C) 2016 Wire Swiss GmbH -->
```

Comment:
@raphaelrobert Looks like we need clarification here. :D We also have (C) 2016 in all other HTML files, like https://github.com/wireapp/wire-webapp/blob/dev/aws/templates/aws/base_error.html